Satellite[] satellites
uint8 version
uint32 gps_tow
uint32 glo_tow
uint32 bds_tow
uint32 qzss_tow
uint16 gps_tow_acc
uint16 glo_tow_acc
uint16 bds_tow_acc
uint16 qzss_tow_acc
uint8 num_sv
uint8 flags